source/fonts/fontawesome-webfont.svg

```diff
@@ -0,0 +1,2671 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Lots of new files are sent to this pool request. Can we make sure that we don't have unused files (fonts, styles, etc.) in the repo?